src/main/webapp/login-status.js

```diff
@@ -49,3 +49,18 @@ function loginInit() {
     });
   });
 }
+
+// Returns whether a user is currently signed in 
+function isUserSignedIn() {
+  const auth2 = gapi.auth2.getAuthInstance();
+  return auth2.isSignedIn.get();
+}
+
+
+function getUserId() {
+  const auth2 = gapi.auth2.getAuthInstance();
+  if (isUserSignedIn()) {
+    var profile = auth2.currentUser.get().getBasicProfile();
+    return profile.getId();
```

Comment:
Google [recommends against](https://developers.google.com/identity/sign-in/web/backend-auth) using this user id but the id_token I get in onSignIn instead.